05_syntax_lab/05.py

```diff
@@ -17,6 +17,21 @@
                 print "Number %d is NOT divided by 13" % (num)
     else:
                 print "Number %d is NOT divided by 7" % (num)
+"""
+
+#New Block (fix with less indentation)
+while True:
+    num = randint(1, 1000000)
+    print "Number is %d" % (num)
+    if num % 7 > 0:
+        continue
+    elif num % 13 > 0:
+        continue
+    elif num % 15 > 0:
+        continue
+    else:

```

Comment:
No need for the else here. The only way to reach this point is to have all other conditions false (since each has a continue)